gen.py

```diff
@@ -80,7 +80,7 @@ def main():
         for cur in range(args.rows):
             line = random.choice(moroccan_cities) + "," + random.choice(fruits_and_vegetables) + "," + str(generate_random_decimal())
             lines += line + '\n'
-            if cur % args.flush:
+            if not cur % args.flush:
```

Comment:
nit: can you make it more explicit
```suggestion
            if cur % args.flush == 0:
```

Reply:
Done